src/vfs/public.c

```diff
@@ -48,12 +50,17 @@ bool vfs_unmount(const char *path) {
 	return false;
 }
 
+
+//not absolute path(this path does not have zip)
 SDL_RWops* vfs_open(const char *path, VFSOpenMode mode) {
```

Comment:
It's more correct to say that this function takes virtual paths, i.e. VFS paths, as opposed to real filesystem paths. "Relative" is confusing, as it means something else, and is only applicable to real filesystem paths (VFS paths are always "absolute", since there is no "working directory" equivalent in the VFS).